packages/changed-elements-react/src/api/VersionCompareTiles.ts

```diff
@@ -173,8 +173,16 @@ export class Provider
   private _eeEmphasized: Id64Set | undefined;
   private _lastJson: EmphasizeElementsProps | undefined;
 
+  /** returns copy of previous ee*/
+  public get lastJson(): EmphasizeElementsProps | undefined {
```

Comment:
Does it make sense for this to be in VersionCompareTiles? It seems more like something the frontstage manager should be doing.